README.md

```diff
@@ -20,14 +20,14 @@ The purpose of the library is to provide a light, simple and general tracing sol
 ## Compromises
 
 * The traces are stored as ASCII arrays in the flash memory (pretty high memory consumption). Therefore, it is not necessary to:
- * encode/decode the trace messages on the fly (this may take too much CPU time) or 
- * have external dev-env dependencies to encode the traces compile time and an external application to decode the traces. 
+  * encode/decode the trace messages on the fly (this may take too much CPU time) or
+  * have external dev-env dependencies to encode the traces compile time and an external application to decode the traces.
 * The group name length is limited to four characters. This makes the lines cleaner and it is enough for most use cases for separating the module names. The group name length may not be suitable for a clean human readable format, but still four characters is enough for unique module names.
-* The trace function uses `stdout` as the default output target because it goes directly to serial port when initialized. 
+* The trace function uses `stdout` as the default output target because it goes directly to serial port in mbed-os.
 * The trace function produces traces like: `[<levl>][grp ]: msg`. This provides an easy way to detect trace prints and separate traces from normal prints (for example with _regex_).
 * This approach requires a `sprintf` implementation (`stdio.h`). The memory consumption is pretty high, but it allows an efficient way to format traces.
 * The solution is not Interrupt safe. (PRs are more than welcome.)
-* The solution is not Thread safe by default. Thread safety for the actual trace calls can be enabled by providing wait and release callback functions that use mutexes defined by the application. 
+* The solution is not Thread safe by default. Thread safety for the actual trace calls can be enabled by providing wait and release callback functions that use mutexes defined by the application.
```

Comment:
Removing the extra whitespace is ok, but I wonder if the T in the first "Thread" should also be made a small one... "Thread" in the sense of an OS thread shouldn't be capitalized, and "Thread" with a capital t is the name of a network protocol (among other things). Of course the context clues should eventually clarify things either way, but still.